resources/views/external-link-confirm.blade.php

```diff
@@ -36,14 +46,20 @@ class="w-full max-w-2xl text-left rounded-xl"
         @endslot
 
         @slot('description')
-            <div class="flex flex-col space-y-4 whitespace-normal">
-                <p>
-                    @lang('generic.external_link_warning')
-                </p>
+            <div class="flex flex-col mt-8 space-y-4 whitespace-normal">
                 <div class="font-semibold text-theme-secondary-900">
                     <x-ark-alert type="warning" :message="$url" />
                 </div>
                 <p>@lang('generic.external_link_disclaimer')</p>
+
+                <x-ark-checkbox
+                    name="terms"
+                    alpine="document.cookie = 'dismiss_external_link_alert=true'"
```

Comment:
```suggestion
                    alpine="document.cookie.match('dismiss_external_link_alert=true') ? document.cookie = 'dismiss_external_link_alert=false' : document.cookie = 'dismiss_external_link_alert=true'"
```
This is do the trick! 😉